packages/react-textarea/src/components/TextArea/useTextAreaStyles.ts

```diff
@@ -0,0 +1,281 @@
+    },
+    ':focus-within': {
+      outlineWidth: tokens.strokeWidthThick,
+      outlineStyle: 'solid',
+      outlineColor: 'transparent',
+    },
+  },
+
+  filledDarker: {
+    backgroundColor: tokens.colorNeutralBackground3,
+    ...shorthands.border(tokens.strokeWidthThin, 'solid', tokens.colorTransparentStrokeInteractive),
+  },
+  filledDarkerInteractive: {
+    ':hover': {
+      ...shorthands.border(tokens.strokeWidthThin, 'solid', tokens.colorTransparentStrokeInteractive),
+    },
+
+    ':active': {
+      ...shorthands.border(tokens.strokeWidthThin, 'solid', tokens.colorTransparentStrokeInteractive),
+    },
+
+    ':focus-within': {
+      ...shorthands.border(tokens.strokeWidthThin, 'solid', tokens.colorTransparentStrokeInteractive),
+    },
+  },
+
+  filledLighter: {
+    backgroundColor: tokens.colorNeutralBackground1,
+    ...shorthands.border(tokens.strokeWidthThin, 'solid', tokens.colorTransparentStrokeInteractive),
+  },
+  filledLighterInteractive: {
+    ':hover': {
+      ...shorthands.border(tokens.strokeWidthThin, 'solid', tokens.colorTransparentStrokeInteractive),
+    },
+
+    ':active': {
+      backgroundColor: tokens.colorNeutralBackground3,
+      ...shorthands.border(tokens.strokeWidthThin, 'solid', tokens.colorTransparentStrokeInteractive),
+    },
+
+    ':focus-within': {
+      backgroundColor: tokens.colorNeutralBackground3,
+      ...shorthands.border(tokens.strokeWidthThin, 'solid', tokens.colorTransparentStrokeInteractive),
+    },
+  },
+
+  outline: {
+    backgroundColor: tokens.colorNeutralBackground1,
+    ...shorthands.border(tokens.strokeWidthThin, 'solid', tokens.colorNeutralStroke1),
+    borderBottomColor: tokens.colorNeutralStrokeAccessible,
+  },
+  outlineInteractive: {
+    ':hover': {
+      ...shorthands.border(tokens.strokeWidthThin, 'solid', tokens.colorNeutralStroke1Hover),
+      borderBottomColor: tokens.colorNeutralStrokeAccessibleHover,
+    },
+
+    ':active': {
+      ...shorthands.border(tokens.strokeWidthThin, 'solid', tokens.colorNeutralStroke1Pressed),
+      borderBottomColor: tokens.colorNeutralStrokeAccessiblePressed,
+    },
+
+    ':focus-within': {
+      ...shorthands.border(tokens.strokeWidthThin, 'solid', tokens.colorNeutralStroke1),
+      borderBottomColor: tokens.colorCompoundBrandStroke,
+    },
+  },
+});
+
+/**
+ * Styles for the textArea slot
+ */
+const useTextAreaStyles = makeStyles({
+  base: {
+    ...shorthands.borderStyle('none'),
+    ...shorthands.margin('0'),
+    backgroundColor: 'transparent',
+    color: tokens.colorNeutralForeground1,
+
+    '::placeholder': {
+      color: tokens.colorNeutralForeground4,
+      opacity: 1,
+    },
+
+    '::selection': {
+      color: tokens.colorNeutralForegroundInverted,
+      backgroundColor: tokens.colorNeutralBackgroundInverted,
+    },
+
+    ':focus-visible': {
+      outlineStyle: 'none', // disable default browser outline
+    },
+  },
+
+  none: {
```

Comment:
I did them this way so I can do: `textAreaStyles[resize]`, but I can change it if it's prefered. It would be cool to be able to do: `textAreaStyles['${resize}Resize']`, but I see how it would look better to add the prefix.